x/gamm/pool-models/balancer/pool_suite_test.go

```diff
@@ -629,109 +806,125 @@ func (suite *KeeperTestSuite) TestBalancerSpotPriceBounds() {
 
 func (suite *KeeperTestSuite) TestCalcJoinPoolShares() {
 	// We append shared calcSingleAssetJoinTestCases with multi-asset and edge
-	// test cases.
+	// test cases defined in multiAssetExactInputTestCases and multiAssetUneverInputTestCases.
+	//
+	// See calcJoinSharesTestCase struct definition for explanation why the
+	// sharing is needed.
+	testCases := append(append(multiAssetExactInputTestCases, multiAssetUnevenInputTestCases...), calcSingleAssetJoinTestCases...)
```

Comment:
Append can take multiple params! (The suggestion is from my a web editor need to double check on the code itself
```suggestion
	testCases := append(multiAssetExactInputTestCases, multiAssetUnevenInputTestCases..., calcSingleAssetJoinTestCases...)
```

Reply:
Noticing that we can't append multiple arrays using `append`, resolving this review